src/compas_fab/backends/ros/backend_features/move_it_plan_motion.py

```diff
@@ -122,10 +122,11 @@ def plan_motion_async(self, callback, errback,
             position=start_configuration.values)
         start_state = RobotState(
             joint_state, MultiDOFJointState(header=header), is_diff=True)
-        attached_collision_meshes = options.get('attached_collision_meshes', [])
-        for acm in attached_collision_meshes:
-            aco = AttachedCollisionObject.from_attached_collision_mesh(acm)
-            start_state.attached_collision_objects.append(aco)
+
+        if options.get('attached_collision_meshes'):
+            for acm in options['attached_collision_meshes']:
+                aco = AttachedCollisionObject.from_attached_collision_mesh(acm)
+                start_state.attached_collision_objects.append(aco)
```

Comment:
that's exactly what it was two or days ago, but i decided to be clever and reduce indentation.

Reply:
;)